lib/index.d.ts

```diff
@@ -725,6 +725,17 @@ export namespace GoogleAdsTypes {
     customData?: string;
   }
 
+  /**
+   * Initialization status of each ad network available to the Google Mobile Ads SDK, keyed by its
+   * GADMAdapter's class name. The list of available ad networks may be incomplete during early
+   * phases of SDK initialization.
+   */
+  export type InitializationStatus = {
+    name: string;
+    description: string;
+    status: 0 | 1;
+  }[];
```

Comment:
An interesting choice to make the type itself an array. Typically the type is not the array, and the return is an array of the type ? Seems more ergonomic to me, as then you can address each array entry as the type, as opposed to some part of the type ?

Reply:
Then the name is maybe not excellent, perhaps GADMAdapterInitializationStatus ? A bit unwieldy, but that captures what it *is* I think?

Reply:
Agree on the ergonomic choice. Will call it "AdapterStatus", that will align with how the types are called in iOS / Android.